src/components/calcite-value-list/calcite-value-list.tsx

```diff
@@ -199,6 +199,56 @@ export class CalciteValueList {
 
   getItemData = getItemData.bind(this);
 
+  keyDownHandler = (event) => {
+    const handleElement = event.composedPath().find((item) => {
+      return item.dataset?.jsHandle;
+    });
+    const valueListElement = event.composedPath().find((item) => {
+      return item.tagName?.toLowerCase() === "calcite-value-list-item";
+    });
+    // Only trigger keyboard sorting when the internal drag handle is focused and activated
+    if (!handleElement || !valueListElement.handleActivated) {
+      return;
+    }
+    const lastIndex = this.items.length - 1;
+    const value = valueListElement.value;
+    const startingIndex = this.items.findIndex((item) => {
+      return item.value === value;
+    });
+    let appendInstead = false;
```

Comment:
nitpick: Maybe the logic below could be in a private method that either returns an index or null if it should append?

Reply:
I created a function called moveNode and added it to the dom utils but I"m not using it right now. One of the particular challenges with this scenario is there are elements that are valid dom children that we have to skip. I'd be down to go over it with you to see if there's something we can't make work. But in its current form it wouldn't skip the header - which is why `sortable.sort` was breaking in Edge.